src/app/clusters/scenes-server/SceneTableImpl.h

```diff
@@ -23,12 +23,13 @@
 #include <lib/support/CommonIterator.h>
 #include <lib/support/PersistentData.h>
 #include <lib/support/Pool.h>
+#include <zap-generated/gen_config.h>
 
 namespace chip {
 namespace scenes {
 
-static constexpr uint16_t kMaxScenesPerFabric   = CHIP_CONFIG_SCENES_MAX_PER_FABRIC;
-static constexpr uint16_t kMaxScenesPerEndpoint = CHIP_CONFIG_MAX_SCENES_PER_ENDPOINT;
+static constexpr uint16_t kMaxScenesPerFabric   = MATTER_SCENES_TABLE_SIZE / 2;
```

Comment:
Why are we removing the ability to configure the per-fabric number separately from the overall endpoint number?

In any case, this does not implement the spec properly as far as I can tell: per spec if the SceneTableSize is 8, say, then the per-fabric number has to be at most 3, but the code here would use 4.

Reply:
Because the Fabric Capacity attribute will be removed from the cluster attributes and I was trying to simplify configuration by making everything go through Zap instead of having to setup the cluster through Zap and `CHIPConfig.h`.

The wording in the spec has indeed change, the limit used to be half of the TableSize... Will update shortly.

Reply:
Updated to match less than half rule in spec.